layouts/shortcodes/pages/compatibility-products.zh.html

```diff
@@ -0,0 +1,48 @@
+<div class="row">
```

Comment:
Rather than create a file for this, we should add new i18n values for the content that needs to be multilingual, and update the base shortcode instead. The new i18n values can be added using a new i18n folder at the root level, and 2 files added to that folder, en.toml and zh.toml. These files will combine with the files we merged into the base theme a week or two ago! More info on how this all works can be found on the [Hugo tutorial/help page for internationalization](https://gohugo.io/content-management/multilingual/#translation-of-strings).

The same page mentioned above will describe the usage of those values as well! We would need something like `{{ i18n "compatability.intro.text" }}`, `{{ i18n "compatability.logo.alt" }}`, and `{{ i18n "compatability.proof-of-compatibility.text" }}`.

This file in particular may need more attention around vendor/product names. I'm not 100% sure how we would handle most of this, as it's IP stuff that's outside of my normal realm.